CHANGELOG.md

```diff
@@ -7,11 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 This repo consists of two packages - the React client and the Python server. A change to either repo should trigger an update in the versions for both to ensure a consistent changelog in this repo.
 
-## [1.20.1] - 2022-03-04
+## [1.20.1] - 2022-03-06
```

Comment:
I think we'll need to add a new entry, rather than change the last one (see [this PR](https://github.com/PolicyEngine/policyengine/pull/533/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed) for an example)

Reply:
It seems I changed the last entry, now I have fixed it, thank you for your information Nikhil!

Reply:
Thanks! I've just updated your entry to use the new format, which hopefully should avoid all the version conflict issues.

Reply:
Thank you!